test/fail_compilation/diag10862.d

```diff
@@ -1,4 +1,4 @@
-/*
+/* REQUIRED_ARGS: -d
```

Comment:
Looks like this is a relic of the old deprecation approach, and it doesn't trigger any deprecation warning anymore.